backend/src/main/java/com/mcmasterbaja/model/AnalyzerEnum.java

```diff
@@ -1,6 +1,6 @@
 package com.mcmasterbaja.model;
 
-public enum AnalyzerType {
+public enum AnalyzerEnum {
```

Comment:
I've already said it many times already but I think that we could come up with a better naming or structure for this. I don't like the type being called Enum